i8_terminal/common/metrics.py

```diff
@@ -117,7 +117,7 @@ def get_period_start_date(period: str) -> str:
     return period_start_date[period]
 
 
-def get_current_metrics_df(tickers: str, metricsList: str) -> Optional[pd.DataFrame]:
+def get_current_metrics_df(tickers: str, metricsList: Union[str, Any, None]) -> Optional[pd.DataFrame]:
```

Comment:
I think Union[str, Any, None] is not meaningful. try to keep this unchanged and check this variable to always be in `str` type when calling this method.